app/src/main/java/com/gruppo4/audioUtility/AudioUtilityManager.java

```diff
@@ -0,0 +1,171 @@
+package com.gruppo4.audioUtility;
+
+/**
+ * @author Francesco Bau'
+ */
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.os.Build;
+import android.os.VibrationEffect;
+import android.os.Vibrator;
+
+import static android.content.Context.AUDIO_SERVICE;
+
+
+public class AudioUtilityManager {
+
+    public static final int MAX_PERCENTAGE = 100;
+    public static final int MIN_PERCENTAGE = 0;
+
+    private static final int VIBRATION_TIME = 500;
+    private static final int VIBRATION_DELAY = 500;
+    private static final int VIBRATION_REPEAT_CODE = 1;
+
+    public enum AUMStream {
+        ALARM,
+        RING,
+        MUSIC
+    }
+
+    /**
+     * @param context The current Context.
+     * @return The current AudioManager instance.
+     */
+    protected static AudioManager getAudioManager(Context context) {
+        return (AudioManager) context.getSystemService(AUDIO_SERVICE);
+    }
+
+    /**
+     * @param context The current Context.
+     * @param stream  The chosen stream (should be ALARM, RING or MUSIC).
+     * @return The current stream Volume (in percentage).
+     */
+    public static int getVolume(Context context, AUMStream stream) {
+        int currentVolume = getAudioManager(context).getStreamVolume(getStream(stream));
+
+        int maxVolume = getMaxVolume(context, stream);
+        return Math.round(MAX_PERCENTAGE * currentVolume / maxVolume);
+    }
+
+    /**
+     * @param context The current Context.
+     * @param stream  The chosen stream (should be ALARM, RING or MUSIC).
+     * @return The maximum stream volume (real value).
+     */
+    private static int getMaxVolume(Context context, AUMStream stream) {
+        return getAudioManager(context).getStreamMaxVolume(getStream(stream));
+    }
+
+    /**
+     * Sets up the stream Volume, given a certain percentage.
+     *
+     * @param context    The current Context.
+     * @param stream     The chosen stream (should be ALARM, RING or MUSIC).
+     * @param percentage Target volume (expressed in %).
+     * @throws IllegalArgumentException if percentage is not between 0 and 100.
```

Comment:
To be coherent with the other style of the comments, start the sentence with the first letter in upper case.